app/(protected)/home/_components/FeatureCard.tsx

```
@@ -0,0 +1,48 @@
+import { ChevronRight } from "lucide-react";
+import Link from "next/link";
+import React from "react";
+
+import { type FeatureCard } from "@/types";
+
+const FeatureCard = ({ Icon, heading, link }: FeatureCard) => {
```

Biome (2.1.2)

[error] Shouldn't redeclare 'FeatureCard'. Consider to delete it or rename it.

'FeatureCard' is defined here:


(lint/suspicious/noRedeclare)